session/handler.go

```diff
@@ -80,6 +80,9 @@ func (h *Handler) whoami(w http.ResponseWriter, r *http.Request, ps httprouter.P
 
 	// s.Devices = nil
 	s.Identity = s.Identity.CopyWithoutCredentials()
+	
+	// Set userId as the X-Authenticated-User header.
+	w.Header().Set("X-Authenticated-User", s.Identity.ID.String())
```

Comment:
I think it would make sense to bind this field to the kratos domain, as `X-Authenticated-User` appears in a several places with different implementations! How about this?

```suggestion
	w.Header().Set("X-Kratos-Authenticated-Identity-Id", s.Identity.ID.String())
```

Alternatively we could also send the identity in the payload:


```suggestion
	w.Header().Set("X-Kratos-Authenticated-Identity", jsonify(s.Identity))
```

Reply:
I agree with the change of header name. 
For me it doesn't matter if the entire Identity is in there but the ID would be enough. Adding the Identity itself would add about 400 bytes of data (which is insignificant of course). But you might help someone else who does want the entire thing. I'll leave the second decision up to you.

Reply:
Ok, let's keep the ID then only :)